packages/xo-server/src/api/xosan.js

```diff
@@ -1542,9 +1542,10 @@ export async function downloadAndInstallXosanPack({ id, version, pool }) {
   const res = await this.requestResource('xosan', id, version)
 
   await xapi.installSupplementalPackOnAllHosts(res)
-  await xapi._updateObjectMapProperty(xapi.pool, 'other_config', {
-    xosan_pack_installation_time: String(Math.floor(Date.now() / 1e3)),
-  })
+  await xapi.pool.update_other_config(
+    'xosan_pack_installation_time',
+    String(Math.floor(Date.now() / 1e3))
+  )
```

Comment:
Aren't you supose to pass an object here instead of two parameters ?

Reply:
`update_*` methods support both APIs, one to set a simple entry and the other to set multiple entries: https://github.com/vatesfr/xen-orchestra/blob/c7c6e0e2ff7a33ff246df1b4c999a126ec17d3fe/packages/xen-api/src/index.js#L1076-L1080